borgcron.sh

```diff
@@ -53,11 +56,7 @@ fi
 export BORG_REPO="$REPOSITORY"
 
 # get passphrase
-if [ -f "$PASSPHRASE_FILE" ]; then
-	export BORG_PASSPHRASE=$( cat "$PASSPHRASE_FILE" )
-else
-	echo "No (valid) passphrase file given."
-fi
+export BORG_PASSPHRASE=$( cat "$PASSPHRASE_FILE" )
```

Comment:
That's okay, however please do not set the variable if the `$PASSPHRASE_FILE` does not exist. It might not be good to set it in that way and try to read a non-existent file.
Short: Just restore the (first part of the) if clause.